server.js

```diff
@@ -27,7 +27,11 @@ app.use(express.json());
 try {
     // Use { force: true } to force db to reset and pick up changes on server restart, remove to surpress
     // this behavior.
-    await sequelize.sync({ force: true });
+    await sequelize.query('SET FOREIGN_KEY_CHECKS = 0').then(data => {
```

Comment:
This is to handle a bug I ran into where Sequelize tries to drop the tables in an invalid order when syncing (it won't drop the "Album" table if the "Like" table still exists with a foreign key dependency on it).  Not sure why this is cropping up now, but this will disable the FK checks while sequelize syncs the DB and turn it back on once it is done.